db/migrate/20230309123141_add_sequence_to_articles.rb

```diff
@@ -0,0 +1,5 @@
+class AddSequenceToArticles < ActiveRecord::Migration[6.1]
+  def change
+    add_column :articles, :sequence, :integer
```

Comment:
Let's use `position` since that's what we have for `categories` to stay consistent in our naming